gallery_dl/extractor/deviantart.py

```diff
@@ -1337,10 +1399,13 @@ def gallery_folders(self, username, offset=0):
         return self._pagination_list(endpoint, params)
 
     @memcache(keyarg=1)
```

Comment:
I didn't realize `memcache` worked across different instances. This fix is kind of hacky though...
```suggestion
    @memcache(keyarg=slice(0, 2))
```